### DISCUSSION TITLE
plugins/plugins/trmenu/

### DISCUSSION THREAD

**Discussion by giscus** [BukkitWiki/BukkitWiki#2] [Announcements]
# plugins/plugins/trmenu/

首页 该 Wiki 是继原 Wiki 补充部分新功能和要点. 简介 TrMenu 是基于 TabooLib 开发，发布于 2019-10-4 的一款高级动态菜单系统 独树一帜的高效、可读 & 可视配置体验 功能强大、极具灵活拓展性 代码开源 截至目前，TrMenu 已经经过三次大版本迭代 本文档仅针对于 TrMenu v3.x（旧版本不受支持，且强烈不...

https://bukkit.wiki/plugins/plugins/trmenu/

**Comment by YsGqHY** [comment]
阔阔太帅了
